service/controller/chart/v1/resource/release/create.go

```diff
@@ -56,8 +56,6 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 			return microerror.Mask(err)
 		}
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("created release %#q", releaseState.Name))
-	} else {
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not create release %#q", releaseState.Name))
```

Comment:
Removing this log line because its not useful. `releasev1.ApplyCreateChange did not create release `